ystia/yorc-a4c-plugin
# Basic implementation of Job spawning shell commands

# Pull Request description

## Description of the change

Added in TOSCA samples, a basic implementation of Job allowing to spawn shell commands,
that could be used to execute basic jobs on infrastructures without a default job scheduler, like Hosts Pools or OpenStack.

### How to verify it

Upload in Alien4Cloud catalog :
  * components at `tosca-samples/org/ystia/yorc/samples/job/noscheduler/components/`
  * topology at `tosca-samples/org/ystia/yorc/samples/job/noscheduler/topologies/chainedJobs`

Create an application from this topology.
Deploy this application on an OpenStack or HostsPool location.

Once the application is deployed, select the `Workflow` tab.
Select the worflow `run`, click on `Launch`.
Check `Job1` is submitted then is run. Check `Job2` is then submitted and run.
 
### Description for the changelog

Add an example of basic job implementation for infrastructures without a default
  job scheduler ([GH-125](https://github.com/ystia/yorc-a4c-plugin/issues/125))

## Applicable Issues

Fixes #125
